src/logistration/LoginPage.jsx

```diff
@@ -13,6 +13,26 @@ import { DEFAULT_REDIRECT_URL } from './data/constants';
 import { loginRequestSelector, thirdPartyAuthContextSelector } from './data/selectors';
 import LoginHelpLinks from './LoginHelpLinks';
 import RedirectLogistration from './RedirectLogistration';
+import LoginFailureMessage from './LoginFailure';
+
+
+const LoginRedirect = (props) => {
```

Comment:
That code is already there. Please rebase.

Reply:
Done!